src/CommunityToolkit.Maui.Core/Services/FragmentLifecycleManager.android.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System.ComponentModel;
+using Android.Content;
+using FragmentManager = AndroidX.Fragment.App.FragmentManager;
+
+namespace CommunityToolkit.Maui.Core.Services;
+
+[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Just realized that devs will need to access this one in order to implement their own `IDialogFragmentService`
```suggestion
```